WORKSPACE

```diff
@@ -179,3 +179,24 @@ load(
 )
 
 apple_support_dependencies()
+
+# For testing the compatibility with a hermetic cc toolchain. Users should not have to enable it.
+
+http_archive(
+    name = "hermetic_cc_toolchain",
+    sha256 = "bd2234acd0837251361be3270d7d3ce599b418be123d902d84762302e31a3014",
+    strip_prefix = "hermetic_cc_toolchain-13c904dce0cb9b6d07f0d557e6ce3cf7013a562e",
+    urls = ["https://github.com/uber/hermetic_cc_toolchain/archive/13c904dce0cb9b6d07f0d557e6ce3cf7013a562e.zip"],
```

Comment:
curious, why not the released version? Any known blockers?

Reply:
Yeah, I need https://github.com/uber/hermetic_cc_toolchain/commit/a9d87b21a5dddd691336c6c0004fa5dcfe5b9b48 to fix some tests.